contributors/guide/README.md

```diff
@@ -12,21 +12,21 @@ description:
 # Getting Started
 
 - [Getting Started](#getting-started)
```

Comment:
As a follow-up, maybe we can remove this section here because it just references itself xD